### DISCUSSION TITLE
Direction of dense layer support

### DISCUSSION THREAD

**Discussion by julian-weinert** [supermerill/SuperSlicer#4420] [Q&A]
I'm a big fan of the "supporting dense layer" option. Unfortunately the direction it's printed in seems to be pretty undeterministic. Sometimes it's aligned with X, sometimes with Y. It appears to be session or model dependant, I still didn't figure it out (it's not influenced by the infill angle at all).

My printer has a part cooling fan in mostly one direction and of course SS always chooses to be perpendicular to that when it's least helpful. I need to align those with my fan blowing direction, so the fan doesn't blow it away for really long bridging layers.

Is there any setting or trick I could use to force SS to print that in a specific direction?

**Comment by supermerill** [comment]
For "supporting dense layer", the angle should be always be 0 (from the printer point of view)

**Comment by julian-weinert** [comment]
I get both, 0 and 90°. It would be useful to be able to have somewhat control over this angle, as I sometimes find myself trying to fix a specific issue with a print
